tests/e2e/scale/test_scale_3_OCS_worker_nodes_and_1500_PVCs.py

```diff
@@ -111,6 +112,71 @@ def test_pv_scale_out_create_pvcs_and_respin_ceph_pods(
         )
 
 
+@scale
+@ignore_leftovers
+@bugzilla("2092737")
+@skipif_external_mode
+@ipi_deployment_required
+@pytest.mark.parametrize(
+    argnames="resource_to_delete",
+    argvalues=[
+        pytest.param(*["noobaa_operator"], marks=[pytest.mark.polarion_id("OCS-4880")]),
+        pytest.param(*["ocs_operator"], marks=[pytest.mark.polarion_id("OCS-4881")]),
+        pytest.param(*["odf_operator"], marks=[pytest.mark.polarion_id("OCS-4882")]),
+        pytest.param(*["operator"], marks=[pytest.mark.polarion_id("OCS-4883")]),
+    ],
+)
+class TestScaleRespinOperatorPods(E2ETest):
+    """
+    Respin Operator pods in cluster with 1500+ PVC & 76 Pods
+    operator ==> OPERATOR_LABEL = "app=rook-ceph-operator"
+    ocs-operator ==> OCS_OPERATOR_LABEL = "name=ocs-operator"
+    odf-operator ==> ODF_OPERATOR_CONTROL_MANAGER_LABEL = "control-plane=controller-manager"
+    noobaa-operator ==> NOOBAA_OPERATOR_POD_LABEL = "noobaa-operator=deployment"
+    """
+
+    def test_respin_operator_pods(
+        self,
+        fioscale,
+        resource_to_delete,
+    ):
+        """
+        Test case to respin operator pods in cluster with scaled PVC+POD's counts
+        """
+
+        # Get info from SCALE_DATA_FILE for validation
+        if os.path.exists(SCALE_DATA_FILE):
+            file_data = templating.load_yaml(SCALE_DATA_FILE)
+            namespace = file_data.get("NAMESPACE")
+            pod_scale_list = file_data.get("POD_SCALE_LIST")
+            pvc_scale_list = file_data.get("PVC_SCALE_LIST")
+        else:
+            raise FileNotFoundError
```

Comment:
can you add a message when you raise the exception. 
for eg., raise FileNotFoundError("scale data files unavailable")

Reply:
updated code.